### PR TITLE
add cask for DesktopUtility

### PR DESCRIPTION
**Pull request by agustif**
#### Adding a new cask
- [X] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [X] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] Commit message includes cask’s name.
- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` left no offenses.
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.

PS. Please excuse me if some what I missed something in the process, this is my first commit but I tried to follow the guidelines, if something is wrong, like maybe missing uninstall_stanza, let me know how to fix it.

### CODE REVIEW

File: Casks/desktoputility.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'desktoputility' do
+  version '2.2'
```
Comment (vitorgalvao) [inline Casks/desktoputility.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

Everything else looks fine. Great first contribution.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in f0c84dda3b347bce5db5950c269a3faf3b9a69cb. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by agustifernandez** [comment]
Thanks adidalal, I couldnt find again the file to apply the fixes, glad someone could help to finish it!

**Pull request closed without merging**